genesis.json

```diff
@@ -258,6 +258,9 @@
     "maximum_tournament_start_time_in_future": 5184000,
     "maximum_tournament_start_delay": 259200,
     "maximum_tournament_number_of_wins": 25,
+    "gpos_period": 15552000,
```

Comment:
Can't change the genesis block!
It would change the chain_id!

If you need additional stuff in the chain parameters, please use the extension field!

Reply:
thank you. moved the parameters to extensions.